octobot_backtesting/api/backtesting.py

```diff
@@ -51,6 +52,10 @@ def create_backtesting(config, data_files) -> Backtesting:
                                           maximum_timestamp=maximum_timestamp)
 
 
+def set_time_updater_interval(interval_in_seconds):
+    TimeManager.TIME_INTERVAL = interval_in_seconds
```

Comment:
What about `TIME_INTERVAL` if we start multiple backtesting ?

Reply:
Right, I didn't change it yet because it's a design problem in TimeManager, since a time manager is specific to a backtesting instance, I think we can rename this variable into DEFAULT_TIME_INTERVAL, and create an instance attribute that will be the used TIME_INTERVAL for a given instance (default being DEFAULT_TIME_INTERVAL). What do you think ?

Reply:
Yes this should be perfect :+1: